src/app/pages/home/home-routing.module.ts

```diff
@@ -7,7 +7,7 @@ import { HomePage } from './home.page';
 const routes: Routes = [
 
 {
-    path:'home',
+    path:'',
```

Comment:
Undo this change. It reads like there are two empty paths. This establishes `/home` for its child nodes: `/home/rolodex`, `/home/profile`, and `/home/exchange`. The empty path below (line 43) directs the user to `/home/rolodex` and the default path for `/home/` (line 36) leads to `rolodex` (which joins the first part to become `/home/rolodex` as well).

Reply:
Two empty paths? But the URLs already have `/home` in them. Admittedly, I don't quite understand why the other routes need to be there.

Reply:
Instead of this...
```
const routes: Routes = [

{
    path:'',
    component: HomePage,
    children: [
      {
        path:'rolodex', loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
      },

      {
        path:'exchange', loadChildren:'./exchange/exchange.module#ExchangePageModule'
      },

      {
        path:'profile',
        children: [
          {
            path:'',
            loadChildren:'./profile/profile.module#ProfilePageModule'
          },
          {
            path:'card-import',
            loadChildren: '../card-import/card-import.module#CardImportPageModule'
          }
        ]
      },

      {
        path: '',
        redirectTo: 'rolodex',
        pathMatch: 'full'
      }
    ]
},
  {
    path: '',
    redirectTo: 'home/rolodex',
    pathMatch: 'full'
  }
];
```
try this...
```
const routes: Routes = [

  {
    path:'rolodex', loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
  },
  {
    path:'exchange', loadChildren:'./exchange/exchange.module#ExchangePageModule'
  },
  {
    path:'profile', loadChildren:'./profile/profile.module#ProfilePageModule',
    children: [
      {
        path:'card-import',
        loadChildren: '../card-import/card-import.module#CardImportPageModule'
      }
    ]
  },
  {
    path: '',
    redirectTo: 'rolodex',
    pathMatch: 'full'
  }
];
```

Reply:
Depending on where `card-import` sits (maybe it's been moved already)

Reply:
Nope. Tabs bar disappears. I'm removing the global path in `app-routing-module.ts` per the suggestion of a tutorial. It seems this delegates the task of specifying the path to the `home-routing-module.ts`.

Reply:
Is there any way to merge them?

```const routes: Routes = [

{
    path:'',
    redirectTo: 'home/rolodex',
    pathMatch: 'full',
    component: HomePage,
    children: [
      {
        path:'rolodex', loadChildren:'./rolodex/rolodex.module#RolodexPageModule'
      },

      {
        path:'exchange', loadChildren:'./exchange/exchange.module#ExchangePageModule'
      },

      {
        path:'profile',
        children: [
          {
            path:'',
            loadChildren:'./profile/profile.module#ProfilePageModule'
          },
          {
            path:'card-import',
            loadChildren: '../card-import/card-import.module#CardImportPageModule'
          }
        ]
      },

      {
        path: '',
        redirectTo: 'rolodex',
        pathMatch: 'full'
      }
    ]
  }
];```

Reply:
This approach didn't work either.